### ISSUE TITLE
Swap sd

### ISSUE THREAD

**Issue by gedi223** [AnalogMan151/NTRBootHax-Pack#2]
I believe @sdswap should be @swapsd in setup script

**Comment by AnalogMan151** [comment]
It’s just a goto label, it can be whatever you want it to be. Feel free to change it if you want.

**Comment by gedi223** [comment]
Yep. I'm stupid.  I updated to the latest godmode9 and luma and was having trouble with the sd swap. Was looking at the script and realized after I posted it's switchsd and the swap sd is just the label.  Tried to delete issue but wouldn't let me.
